proto/iotics/api/search.proto

```diff
@@ -111,7 +111,7 @@ message SearchResponse {
   // Search response twin details.
   message TwinDetails {
```

Comment:
this is the "Twin" object really - should we not have it in common and shared across all the services?

Reply:
I agree, but this is starting to go beyond the scope of this change. 
Updating these (twin/feed) objects and sharing them will ripple down to the sparql level.
And whilst this is going to be a breaking change anyway, I think we should hold off for now.
The ID updates are a stepping stone to this and hopefully prove useful.